kpi/paginators.py

```diff
@@ -151,6 +151,14 @@ def get_count(self, queryset):
         return super().get_count(queryset)
 
 
+class LimitStartPagination(LimitOffsetPagination):
+    """
+    A pagination class that supports '?limit=<count>&start=<offset>' query parameters.
+    """
+
+    offset_query_param = 'start'
+
+
```

Comment:
is it really needed? we can tell the product team to use `offset` instead of `start` to avoid having two almost identical class.